todo.js

```diff
@@ -85,9 +85,22 @@ function descriptionChange(e) {
             if (tasksDom.length === createdTasks.length) {
                 createdTasks[i].description = e.currentTarget.value;
                 localStorage.setItem('saveTasks', JSON.stringify(createdTasks));
+            } else {
```

Comment:
Use `return` after `localStorage.setItem('saveTasks', JSON.stringify(createdTasks));` and remove `else` block